tests/sentry/utils/test_pickle_protocol.py

```diff
@@ -1,29 +0,0 @@
-import pickle
-from pickle import PickleBuffer, PickleError
-
-import pytest
-
-from sentry.testutils.cases import TestCase
-
-
-class PickleProtocolTestCase(TestCase):
-    """
-    At the time of adding this test we still monkey patch `pickle` and hardcode the protocol to be 2.
-    For legacy reasons see `src/sentry/monkey/pickle.py`.
-
-    This test is for a change that's being made to allow explicitly passing a newer protocol to
-    pickle. If we remove the monkey patching to pickle there is no longer a need for this test.
-
-    """
-
-    def test_pickle_protocol(self):
-        data = b"iamsomedata"
-
-        pickled_data = PickleBuffer(data)
-        with pytest.raises(PickleError) as excinfo:
-            result = pickle.dumps(pickled_data)
```

Comment:
this test actually passed because pickle still defaults to protocol 4 -- but since we're no longer patching dumps I removed it anyway